### PR TITLE
Fix docker labels (fixes #34)

### LINKED ISSUE #34: Add appropriate object labels
https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#label

### PR DESCRIPTION
**Pull request by JonZeolla**
# Contributor Comments
Fix docker labels and add an OCI-compatible (i.e. incomplete) annotations linter for `Dockerfile` `LABEL`s.

## Pull Request Checklist
Thank you for submitting a contribution to `easy_infra`!

In order to streamline the review of your contribution we ask that you review and comply with the below requirements:
 - [X] Ensure that `make build` completes successfully
 - [X] Rebase your branch against the latest commit of the target branch, which likely should be `master`
 - [X] If there is an issue associated with your Pull Request, align your PR title with [this documentation](https://help.github.com/en/articles/closing-issues-using-keywords)

### CODE REVIEW

**Review by pb-seiso — APPROVED** [review]
I'm not previously familiar with GitHub actions workflow files but what you're doing seems clear. OCI format looks good too.

File: .github/workflows/docker-ci.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
```
Comment (JonZeolla) [inline .github/workflows/docker-ci.yml:12]: This doesn't make sense can you please explain?

File: .github/workflows/docker-ci.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  lint:
```
Comment (JonZeolla) [inline .github/workflows/docker-ci.yml:13]: Other thing

File: .github/workflows/docker-ci.yml (hunk 1 above)
Comment (JonZeolla) [inline .github/workflows/docker-ci.yml:12]:
```suggestion
# Leave a comment
jobs_to_consider:
```

Comment (JonZeolla) [reply .github/workflows/docker-ci.yml:12]: Uhh not sure this is valuable

**Review by JonZeolla — COMMENTED** [review]
Some feedback for you; overall looking pretty good

File: .github/workflows/docker-ci.yml (hunk 1 above)
Comment (JonZeolla) [reply .github/workflows/docker-ci.yml:12]: yeah i disagree

**Comment by JonZeolla** [comment]
@pb-seiso this is ready for you again, added some very basic rules for the `Dockerfile` linting.

**Comment by JonZeolla** [comment]
This is rebased on `master` and ready to go

**Review by pb-seiso — APPROVED** [review]
Reviewed and ran a few checks on the value regexes to test the label linter. LGTM

**Pull request merged**